### ISSUE TITLE
XML encoding should be 'UTF-8' rather than 'utf8'

### ISSUE THREAD

**Issue by rick-man** [13rac1/scfbuild#9]
Hi there,

I'm from the Direct2D/DirectWrite team at Microsoft. As you may know, we've recently [added support](https://msdn.microsoft.com/en-us/library/windows/desktop/hh802478%28v=vs.85%29.aspx) for OpenType SVG fonts in Windows.

I noticed an issue in a recent version of [LapisLegit](https://github.com/RoelN/LapisLegit), which is built with `scfbuild`.

The `add_color_svg` method of the `Builder` class currently passes `encoding='utf8'` to `ET.tostring`. This is an invalid encoding string, and strict XML parsers might not permit it. The correct string is `'UTF-8'`.

For more info:
- http://www.w3.org/TR/2006/REC-xml11-20060816/#NT-EncodingDecl
- https://www.iana.org/assignments/character-sets/character-sets.xhtml

**Comment by 13rac1** [comment]
Good to hear from you! Yes, I know all about color font support in Windows, I've just fixed SCFBuild to output SVGs with the correct transform. I still need to release new versions of my emoji fonts, but have been busy... 

I ran http://xerographer.github.io/reinebow/ through `ttx` and confirmed the XML encoding is wrong:

``` xml
      <![CDATA[<?xml version='1.0' encoding='utf8'?>
```

I know what the encoding is supposed to be when I type it myself! Argh...

Checking the source for ElementTree: https://hg.python.org/cpython/file/2.7/Lib/xml/etree/ElementTree.py#l810 

``` python
        elif xml_declaration or (xml_declaration is None and
                                 encoding not in ("utf-8", "us-ascii")):
```

> XML processors SHOULD match character encoding names in a case-insensitive way

ElementTree seems to be looking for lowercase, but the recommendation per the spec is uppercase. I'll add uppercase.
